src/App.js

```diff
@@ -21,7 +21,13 @@ class App extends Component {
   }
 
   componentWillMount() {
-    this.getUserData();
+    this.interval = setInterval(() => {
+      this.getUserData();
```

Comment:
Should you verify if the user returned by doppler is the same that the user of the JWT token?

Reply:
i added now

Reply:
What to do if it is different? we should make honor to _Doppler MVC_ response or _JWT_ token?